src/web/modules/stripe/basic/test-account.http.ts

```diff
@@ -0,0 +1,141 @@
+import { StripePerson } from '../model/person.model'
+
+const Stripe = require('stripe-latest')
+const { StripeError } = Stripe.errors
+const STRIPE_TEST_ACCOUNT_API_KEY: string = process.env.STRIPE_TEST_ACCOUNT_API_KEY || ''
+const stripe = new Stripe(STRIPE_TEST_ACCOUNT_API_KEY, {
+    apiVersion: '2020-08-27',
+})
+
+if (config.server.HTTPS_PROXY) {
+    // @ts-ignore
+    stripe.setHttpAgent(new HTTPSProxyAgent(config.server.HTTPS_PROXY))
+}
+
+function validateRequest(currentPspTestAccountStage: string) {
+    if (currentPspTestAccountStage === 'CREATED') {
+        throw new CustomValidationError('Stripe test account has already been created for service')
+    }
+    if (currentPspTestAccountStage !== 'REQUEST_SUBMITTED') {
+        throw new CustomValidationError('Service has not completed a request for Stripe test account')
+    }
+}
+
+const createTestAccount = async function createTestAccount(req: Request, res: Response): Promise<void> {
+    if (!STRIPE_TEST_ACCOUNT_API_KEY) {
+        throw new CustomValidationError('Stripe test API Key was not configured for this Toolbox instance')
+    }
+
+    const serviceExternalId = req.query.service as string
+    const service: Service = await AdminUsers.service(serviceExternalId)
+
+    validateRequest(service.current_psp_test_account_stage)
+
+    const context: {
+        systemLinkService: string; serviceName: string;
+        csrf: string;
+        flash: object
+    } = {
+        systemLinkService: serviceExternalId,
+        serviceName: service.service_name.en,
+        csrf: req.csrfToken(),
+        flash: req.flash()
+    }
+
+    return res.render('stripe/basic/confirm-create-test-account', context)
+}
+
+const createTestAccountConfirm = async function createTestAccountConfirm(req: Request, res: Response, next: NextFunction): Promise<void> {
+    const { systemLinkService } = req.body
+    let service: Service
+
+    try {
+        service = await AdminUsers.service(systemLinkService)
+
+        validateRequest(service.current_psp_test_account_stage)
+
+        const stripeAccount = await createStripeTestAccount(service.service_name.en)
+
+        const account: CardGatewayAccount = await createTestGatewayAccount(systemLinkService, service.service_name.en, stripeAccount.id)
+        const gatewayAccountIdDerived = String(account.gateway_account_id)
+        res.render('gateway_accounts/createSuccess', {
+            account: account,
+            linkedService: systemLinkService,
+            gatewayAccountIdDerived,
+            provider: 'stripe',
+            isLive: false,
+            isStripe: true,
+            stripeConnectAccountId: stripeAccount.id,
+            selfServiceBaseUrl: config.services.SELFSERVICE_URL
+        })
+    } catch (error) {
+        if (error instanceof StripeError) {
+            console.log(error)
+            console.log(error.message)
+            logger.error(`Stripe Error - ${error.stack}`)
```

Comment:
Recommend not logging the stack directly unless it's absolutely needed, if there's a completely unknowable error it will go to `next` which will decide if it should show the stack or not depending on the environment
```suggestion
            logger.error(`Stripe Error - ${error.message}`)
```